<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameWork</class>
 <widget class="QWidget" name="FrameWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,14,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="layoutTitleBar"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutCenter"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutStatusBar"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
